src/types/events/base-events.ts

```diff
@@ -769,13 +809,51 @@ export interface SharedChannelInviteDeclined {
   declining_user: SharedChannelUserItem;
   event_ts: string;
 }
+// for backward-compatibility
+export type SharedChannelInviteDeclined = SharedChannelInviteDeclinedEvent;
 
-export interface SharedChannelInviteReceived {
+export interface SharedChannelInviteReceivedEvent {
   type: 'shared_channel_invite_received';
   invite: SharedChannelInviteItem;
   channel: SharedChannelItem;
   event_ts: string;
 }
+// for backward-compatibility
+export type SharedChannelInviteReceived = SharedChannelInviteReceivedEvent;
+
+export interface SharedChannelInviteRequestedEvent {
+  type: 'shared_channel_invite_requested';
+  actor: {
+    id: string;
+    name: string;
+    is_bot: boolean;
+    team_id: string;
+    timezone: string;
+    real_name: string;
+    display_name: string;
+  };
+  channel_id: string;
+  event_type: 'slack#/events/shared_channel_invite_requested';
```

Comment:
I've tested all aboev app_ events but I haven't checked how this goes not on automation platform. This may be nullable or optional.